docs/configuration/settings.rst

```diff
@@ -447,7 +447,131 @@ Company settings
+
+AMQP
+----
+The AMQP transport requires the AMQP PHP extension and a running AMQP service like `RabbitMQ <https://www.rabbitmq.com/>`_. See the screenshot below for an example of the configuration.
+
+.. image:: images/queue-amqp.png
+  :width: 600
+  :alt: Example of AMQP transport
+
+See `https://symfony.com/doc/5.4/messenger.html#amqp-transport <https://symfony.com/doc/5.4/messenger.html#amqp-transport>`_ for the complete list of configuration options.
+
+Beanstalkd
+----------
+The Beanstalkd transport requires a running `Beanstalkd <https://beanstalkd.github.io/>`_ service and a composer dependency installed via ``composer require symfony/beanstalkd-messenger``.
+After installing the composer dependency, you can fill in the configuration as follows.
+
+.. image:: images/queue-beanstalkd.png
+  :width: 600
+  :alt: Example of Beanstalkd transport
+
+See `https://symfony.com/doc/5.4/messenger.html#beanstalkd-transport <https://symfony.com/doc/5.4/messenger.html#beanstalkd-transport>`_ for the complete list of configuration options.
+
+Amazon SQS
+----------
+The Amazon SQS transport is ideal when your Mautic instance is hosted on AWS. You will need to install a composer dependency via ``composer require symfony/amazon-sqs-messenger``. See the example of the configuration below.
+
+.. image:: images/queue-amazon-sqs.png
+  :width: 600
+  :alt: Example of Amazon SQS transport
+
+See `https://symfony.com/doc/5.4/messenger.html#amazon-sqs <https://symfony.com/doc/5.4/messenger.html#amazon-sqs>`_ for the complete list of configuration options.
+
+How to consume messages from the queue
+======================================
+
+To start consuming the messages from the queue, you need to run the following Symfony command.
+
+.. code-block::
+
+    php bin/console messenger:consume email
+
+If you don't use Kubernetes in your environment, use a process manager like Supervisor or systemd to keep your worker(s) running. More on this at
+`https://symfony.com/doc/5.4/messenger.html#consuming-messages-running-the-worker <https://symfony.com/doc/5.4/messenger.html#consuming-messages-running-the-worker>`_
+
+Advanced setting
+================
+
+Retry strategy
+--------------
+
+When the processing of a message fails, the message is sent back to the queue for another try. You can adjust this behaviour in this section.
+See `https://symfony.com/doc/5.4/messenger.html#retries-failures <https://symfony.com/doc/5.4/messenger.html#retries-failures>`_ for more details.
+
+The screenshot below shows the default values.
+
+.. image:: images/queue-retry-strategy.png
+  :width: 600
+  :alt: Retry strategy defaults
+
+Queue for failures
+------------------
+
+If a message fails all its retries, it's discarded by default. To avoid this happening, you can optionally configure a queue for failures.
```

Comment:
```suggestion
If a message fails all its retries, it's discarded by default. To avoid this happening, you can optionally configure a Queue for failures.
```